tests/test_indent_dedent.py

```diff
@@ -198,28 +198,6 @@ def check(filename, input_commands, output):
     return check
 
 
-def test_markdown_autoindent(check_autoindents):
-    check_autoindents(
```

Comment:
Can we keep this, but just modify it so that it passes?

Reply:
I got rid of it because it felt redundant with all the markdown tests, but I can look back and see if there is a way to make that happen